src/ComputeSharp/Graphics/Resources/Debug/BufferDebugView{T}.cs

```diff
@@ -20,7 +20,7 @@ public BufferDebugView(Buffer<T>? buffer)
         {
             var items = GC.AllocateUninitializedArray<T>(buffer.Length);
 
-            buffer.CopyTo(ref items[0], buffer.Length, 0);
+            buffer.CopyTo(ref items[0], 0, buffer.Length);
```

Comment:
Were these the types of breaking changes you were talking about? Bit confused on why the order needed to change here? (Probably as looking at code on Github is hard... 😋)

Reply:
Yup 😄
The reason for the change is the same I mentioned above: it allows all APIs to be consistent now across different overloads. Now regardless of whatever method you're calling you _always_ have source parameters first, then destination parameters, then count, both when copying to and when copying from. Makes it much harder to accidentally swap the source and destination parameters when doing copies.